oss-internship-2020/libarchive/ld_preload_example/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_library(minitar_lib SHARED
+  minitar.cc
+  minitar.h
+)
+
+target_link_libraries(minitar_lib PRIVATE
+  archive
+)
+
+add_executable(minitar_original
+  minitar_main.cc
+)
+
+target_link_libraries(minitar_original PRIVATE
+    minitar_lib
+)
+
+
+#===========
+
```

Comment:
Please remove. If you want to keep a comment, please omit the blank lines, format it with one space and comment on what this section is for.

Same below.